cmd/version.go

```diff
@@ -81,3 +87,38 @@ func getPluginVersions(opts Options) []string {
 
 	return versions
 }
+
+// Checks GitHub releases and prints new version, if current version is outdated.
+// requires GitHub releases to follow semver.
+func (cli *CLI) printLatestReleaseVersion() {
```

Comment:
There should be a way to disable this, e.g., for users who want to run TFLint offline. See https://developer.hashicorp.com/terraform/cli/commands#upgrade-and-security-bulletin-checks.